onnx/version_converter.py

```diff
@@ -168,7 +200,7 @@ def convert_version(model: ModelProto, target_version: int) -> ModelProto:
         )
     model_str = model.SerializeToString()
     converted_model_str = C.convert_version(model_str, target_version)
-    return load_from_string(converted_model_str)
+    return _deserialize(converted_model_str, ModelProto())
```

Comment:
May I understand why using _deserialize instead of load_from_string? (They seem identical)

Reply:
I was just trying to avoid a circular dependency. Load from string is defined in `__init__`, but init tried to import a module that needed `version_converter`, which required `load_from_string`. So I added its own `_deserialize` function to `version_converter`, so that it can be imported from submodules that are imported by `__init__`.

Reply:
We can probably extract `_deserialize` to another (internal ie prepended with `_`) module that is not `__init__`?

Reply:
At the same time if you import just `onnx` and use `load_from_string` as `onnx.load_from_string` here I don't think there will be circular dependencies

Reply:
Good point, done.